sdk/README.md

```diff
@@ -12,142 +12,149 @@
+```sh
+sudo apt update
+sudo apt install -y libtool automake autoconf curl build-essential python2-minimal
+```
 
-To create a blockchain application, you need to provide an entry point of your application (like `index.js`) and set-up your network by using the modules of Lisk SDK.
+##### macOS
 
-It is quite simple to have a working blockchain application, mirroring the configuration of the existing Lisk network. This can be done by copying the following three lines of code to your `index.js`:
+Ensure that [Homebrew](https://brew.sh/) is installed.
 
-```js
-const { Application, genesisBlockDevnet, configDevnet } = require('lisk-sdk');
+```sh
+brew install autoconf automake libtool python2
+```
 
-const { app } = Application.defaultApplication(
-	genesisBlockDevnet,
-	configDevnet
-);
+### Initializing a new blockchain client
 
-app
-	.run()
-	.then(() => app.logger.info('App started...'))
-	.catch((error) => {
-		console.error('Faced error in application', error);
-		process.exit(1);
-	});
+Execute the `lisk init` command of Lisk Commander as shown below, to bootstrap a default blockchain client in the current folder.
+
+```sh
+lisk init
 ```
 
-After that you can start the application by:
+As a result of executing the `init` command, you will be asked for the _Name_, _ChainID_, _Description_, _Author_, and _License_ of the new blockchain client.
 
 ```
-node index.js
+Using template "lisk-ts"
+Initializing git repository
+Updating .liskrc.json file
+Creating project structure
+? Application name my_client
+? Chain ID in hex representation. ChainID must be 4 bytes (8 characters) 12345678
+? Application description A simple blockchain client for my Web3 application
+? Author XYZ
+? License ISC
+```
+
+Next, all the required files are created by Lisk Commander.
+
+It is now already possible to start the newly bootstrapped blockchain client with default configurations for a local Devnet:
+
+```sh
+./bin/run start
 ```
 
 ### Configure your blockchain parameters
 
-You can also define your blockchain application parameters such as `blockTime`, `maxTransactionsSize` and more with an optional configurations object.
-
-```js
-const { app } = Application.defaultApplication(genesisBlockDevnet, {
-    genesisConfig: {
-      blockTime: 5,
-      maxTransactionsSize: 100 * 1024,
-      minRemainingBalance: "5000000",
-	  activeValidators: 101,
-	  standbyValidators: 2,
-	  validatorListRoundOffset: 2
-    },
-    ...
-});
+The blockchain client offers a variety of configuration options such as `blockTime`, `maxTransactionsSize` and more.
+
+For a complete overview of the client configuration options, please check out the [SDK configuration reference](https://lisk.com/documentation/lisk-sdk/v6/config.html) and the [Blockchain client configuration](https://liskhq.github.io/lisk-docs/beta/build-blockchain/configuration.html) guide.
```

Comment:
`Blockchain client configuration` points to staging docs. Shouldn't this point to the main site?